instructor/common.py

```diff
@@ -665,11 +665,21 @@ def western_blot_band_size(request):
     pk = request.session['assignment_id']
     assignment = models.Assignment.objects.get(id=pk)
     wb, created = models.WesternBlot.objects.get_or_create(assignment=assignment)
-
+    exclude_fields = ['primary', 'secondary', 'western_blot']
+    lysate_types = {
+        'has_whole_cell_lysate': 'wc_weight',
+        'has_nuclear_fractination': 'nuc_weight',
```

Comment:
It looks like `fractionation` is misspelled here and in the model, not sure if it's worth the effort to fix though